topi/python/topi/intel_graphics/conv2d.py

```diff
@@ -64,7 +59,12 @@ def _alter_conv2d_layout(attrs, inputs, tinfos, F):
     new_attrs = {k: attrs[k] for k in attrs.keys()}
     new_attrs['kernel_layout'] = 'OIHW%do' % (oc_bn)
 
-    return sym.contrib.conv2d_NCHWc(*copy_inputs, **new_attrs)
+    if F == sym:
+        out = F.contrib.conv2d_NCHWc(*copy_inputs, **new_attrs)
+    else:
+        out = F.nn.contrib_conv2d_nchwc(*copy_inputs, **new_attrs)
```

Comment:
does it work? `Conv2dAttr` contains `layout` in nnvm while in Relay the name is `data_layout` though

Reply:
It seems working for me.  `relay.nn.contrib_conv2d_nchwc`

Reply:
oh yes, because it is copy, relay copies to relay, nnvm copies to nnvm.